.gitignore

```diff
@@ -1,95 +1,91 @@
-# logs
-/logs
-*.log
-
-# dependencies
+# Dependencies #
+################
 vendor/
 node_modules/
 
-# build directories
+# Build directories #
+#####################
+.sass-cache
 /_deploy_*/
 
-# vvv
+# VVV #
+#######
 /log/
 /wp-config-vvv.php
 
-# plugins
-/content/mu-plugins/*
-!/content/mu-plugins/autoload.php
-!/content/mu-plugins/fix-annoyances.php
-!/content/mu-plugins/fix-svg.php
-!/content/mu-plugins/fix-https.php
-!/content/mu-plugins/loader.php
-/content/plugins/*
+# Web Server #
+##############
+.htaccess
 
-# themes
-/content/themes/*
+# WP Plugins #
+##############
+/wp-content/mu-plugins/*
+!/wp-content/mu-plugins/autoload.php
+!/wp-content/mu-plugins/fix-annoyances.php
+!/wp-content/mu-plugins/fix-svg.php
+!/wp-content/mu-plugins/fix-https.php
+!/wp-content/mu-plugins/loader.php
+/wp-content/plugins/*
+
+# WP Themes #
+#############
+/wp-content/themes/*
 # Include your theme like this:
-#!/content/themes/your-theme
+#!/wp-content/themes/your-theme
 
-# jetbrains
+# IDEs #
+########
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+*.un~
+Session.vim
+.netrwhist
+*~
 .idea/
 .idea_modules/
 *.ipr
 *.iws
 *.iml
 atlassian-ide-plugin.xml
 com_crashlytics_export_strings.xml
-
-# vim
-[._]*.s[a-w][a-z]
-[._]s[a-w][a-z]
-*.un~
-Session.vim
-.netrwhist
-*~
-
-# sublime text
 *.sublime-workspace
+.vscode
 
-# system files
-.DS_Store
-._*
-.AppleDouble
-.LSOverride
-.Spotlight-V100
-.Trashes
-*.bak
-*.swp
-*.orig
-Thumbs.db
-
-# vcs
+# VCS #
+#######
 .svn
 .cvs
 
-# wordpress core
+# WordPress #
+#############
 /core/
+wp-config.php
+wp-content/uploads/
+wp-content/blogs.dir/
+wp-content/upgrade/*
+wp-content/backup-db/*
+wp-content/advanced-cache.php
+wp-content/wp-cache-config.php
+wp-content/cache/*
+wp-content/cache/supercache/*
+wp-content/languages/
 
-# wordpress specific
-content/uploads/
-content/blogs.dir/
-content/upgrade/*
-content/backup-db/*
-content/advanced-cache.php
-content/wp-cache-config.php
-content/cache/*
-content/cache/supercache/*
-content/languages/
-
-# wpengine specific
+# WP Engine #
```

Comment:
We should probably remove this section, it seems pointless to maintain something that is this specific on something that should be generic.

Reply:
Please review. I've grouped some of the rules.